src/app/tests/suites/TestDiscovery.yaml

```
@@ -31,9 +31,13 @@ config:
         defaultValue: 5
 
 tests:
-    - label: "Reboot target device"
+    - label: "Stop target device"
       cluster: "SystemCommands"
-      command: "Reboot"
+      command: "Stop"
+
+    - label: "Start target device"
```

```suggestion
    - label: "Start target device with the provided discriminator for basic commissioning advertisement"
```
and ideally something similar in the other places where we do this.